### PR TITLE
Fixing script to be able to skip logging in as admin, if needed

### PR DESCRIPTION
**Pull request by mohammedzee1000**
This PR makes figuring out KUBEADMIN information optional
Needed to fix OpenShift CI
Signed-off-by: Mohammed Zeeshan Ahmed <mohammed.zee1000@gmail.com>

### CODE REVIEW

**Comment by mohammedzee1000** [comment]
Needs https://github.com/openshift/release/pull/3089

File: scripts/configure-installer-tests-cluster.sh [hunk 1]
```diff
@@ -15,19 +15,25 @@ export KUBECONFIG=${KUBECONFIG:-"${DEFAULT_INSTALLER_ASSETS_DIR}/auth/kubeconfig
 # List of users to create
 USERS="developer odonoprojectattemptscreateproject odosingleprojectattemptscreate odologinnoproject odologinsingleproject1"
 
-# Check if nessasary files exist
-if [ ! -f $KUBEADMIN_PASSWORD_FILE ]; then
-    echo "Could not find kubeadmin password file"
-    exit 1
-fi
+# Attempt resolution of kubeadmin, only if a CI is set
+if [ -z $CI ]; then
```
Comment (kadel) [inline scripts/configure-installer-tests-cluster.sh]: Can you add some information why is this needed, and how it is used?

Comment (mohammedzee1000) [reply scripts/configure-installer-tests-cluster.sh]: SHort story apparently OpenShift CI has black magic to export kubeconfig and login as kubeadmin. Hence this needs to be skipable

Comment (mohammedzee1000) [reply scripts/configure-installer-tests-cluster.sh]: Done

File: scripts/configure-installer-tests-cluster.sh [hunk 2]
```diff
@@ -8,26 +8,32 @@ HTPASSWD_SECRET="htpasswd-secret"
 DEFAULT_INSTALLER_ASSETS_DIR=${DEFAULT_INSTALLER_ASSETS_DIR:-$(pwd)}
 KUBEADMIN_USER=${KUBEADMIN_USER:-"kubeadmin"}
 KUBEADMIN_PASSWORD_FILE=${KUBEADMIN_PASSWORD_FILE:-"${DEFAULT_INSTALLER_ASSETS_DIR}/auth/kubeadmin-password"}
-SLEEP_AFTER_SECRET_CREATION=${SLEEP_AFTER_SECRET_CREATION:-18}
+SLEEP_AFTER_SECRET_CREATION=${SLEEP_AFTER_SECRET_CREATION:-20}
```
Comment (amitkrout) [inline scripts/configure-installer-tests-cluster.sh]: ```SLEEP_AFTER_SECRET_CREATION``` not in use. Please remove this

Comment (mohammedzee1000) [reply scripts/configure-installer-tests-cluster.sh]: Done

File: scripts/configure-installer-tests-cluster.sh [hunk 3]
```diff
@@ -75,10 +77,17 @@ spec:
...
+    if [ $? -eq 0 ]; then
+        break
+    fi
+    sleep 2
+done
+
 oc new-project myproject
 sleep 4
+
+make bin
```
Comment (amitkrout) [inline scripts/configure-installer-tests-cluster.sh]: Why you add make bin at the end of the script. Are you copying it to the prow setup after that through release script ?

Comment (mohammedzee1000) [reply scripts/configure-installer-tests-cluster.sh]: Will remove once make bin is added to tests
this is a TODO

Comment (amitkrout) [reply scripts/configure-installer-tests-cluster.sh]: https://github.com/openshift/release/pull/3100

File: scripts/configure-installer-tests-cluster.sh (hunk 1 above)
Comment (kadel) [reply scripts/configure-installer-tests-cluster.sh]: This is the information that we need put into the comment. No one will know what this is in a few weeks :-(
